integration

```diff
@@ -688,6 +688,7 @@
   "mathieu-mp/homeassistant-intex-spa",
   "MatthewOnTour/BUT_blinds_time_control",
   "mattrayner/pod-point-home-assistant-component",
+  "mawaqit/home-assistant",
```

Comment:
<https://github.com/mawaqit/home-assistant/tree/v3.0.0/custom_components>
As pr the docs, this directory should only contain one sub-directory.